src/index.js

```diff
@@ -159,19 +183,19 @@ export function ExportExtendedPublicKey({
                                         }) {
   switch (keystore) {
     case TREZOR:
-      return new TrezorExportExtendedPublicKey({
+      return new TrezorExportExtendedPublicKey({bip32Path,
         network,
-        bip32Path,
-        includeXFP,
-      });
+        includeXFP});
     case HERMIT:
       return new HermitExportExtendedPublicKey({bip32Path});
+    case COLDCARD:
+      return new ColdcardExportExtendedPublicKey({bip32Path,
```

Comment:
I'm surprised the linter isn't picking these up but the props being stacked should be formatted as they are further down in line 288

```
      return new ColdcardSignMultisigTransaction({
        network,
        inputs,
        outputs,
        bip32Paths,
        psbt,
      });
```